src/App/ViewModels/JobsListViewModel.cs

```diff
@@ -16,10 +17,15 @@ class JobsListViewModel : INotifyPropertyChanged
     {
         //public AppConstant ServiceConstant = new AppConstant();
         public JobListModel JobsList { get; set; }
-        public JobsListViewModel()
+        public Command CallDetailView { get; set; }
```

Comment:
The name of the property should end in "Command" ->i.e.:  CallDetailScreenCommand

Reply:
corregido